packages/teams-js/src/internal/videoPerformanceMonitor.ts

```diff
@@ -31,9 +31,11 @@ export class VideoPerformanceMonitor {
 
   private performanceStatistics: VideoPerformanceStatistics;
 
-  public constructor(private reportPerformanceEvent: (actionName: string, args: unknown[]) => void) {
+  public constructor(
+    private reportPerformanceEvent: (apiVersionTag: string, actionName: string, args: unknown[]) => void,
+  ) {
     this.performanceStatistics = new VideoPerformanceStatistics(VideoPerformanceMonitor.distributionBinSize, (result) =>
-      this.reportPerformanceEvent('video.performance.performanceDataGenerated', [result]),
+      this.reportPerformanceEvent('v0_apiVersionTag_Sample', 'video.performance.performanceDataGenerated', [result]),
```

Comment:
I don't think `videoPerformanceMonitor` needs to know what is apiVersionTag when reporting performance event, instead, you may wrap the function when creating instance of `VideoPerformanceMonitor` like:

```
new VideoPerformanceMonitor(
  (actionName: string, args: unknown[]) => sendMessageToParent('v0_apiVersionTag_Sample', actionName, args)
)

Reply:
That's what I expect. But we would like to set the first argument / apiVersionTag to be required so that we could force developers to label apiVersionTag if it is needed for telemetry.

My concern is that, if we use new function (sendMessageToParent) with additional parameter, which is apiVersionTag, to communicate between app and host sdk or app and host, do you think it will trigger any error or exceptions based on your experience?

Reply:
> if we use new function (sendMessageToParent) with additional parameter, which is apiVersionTag, to communicate between app and host sdk or app and host, do you think it will trigger any error or exceptions based on your experience?

It really depends on how you implement the new function, especially, are you going to update the hub sdk as well? Our use case is to use this function to send message, and it will be received by hub sdk and forward to our front side code, as long as this process is not changed, I am good with the change.

Reply:
Yes, we will also update corresponding change in Hub SDK side and we made demo and validated the idea that the change here will have no harm to Hub SDK / host even if they don't change anything. So I think I have gotten my answer. Thanks again! Appreciated.